src/themes/default/components/core/blocks/Footer/Footer.vue

```diff
@@ -170,6 +170,9 @@ $gray-secondary: map-get($colors, gray-secondary);
       margin-left: 20px;
     }
   }
+  a {
```

Comment:
Please use .no-underline class instead of this

Reply:
I dont understand... It should be underlined :) In previous commits i deleted this accidentally, cuz thought it's a bug of double underlining on hover.
https://github.com/DivanteLtd/vue-storefront/blob/master/src/themes/default/components/core/blocks/Footer/Footer.vue#L172

Reply:
ahh sorry man ;-)

Reply:
![image](https://user-images.githubusercontent.com/11723092/35348950-b1e5c6e4-014a-11e8-8481-2f2db95e5d5e.png)
Cant see  2.
Same for 3.
![image](https://user-images.githubusercontent.com/11723092/35349031-e56309dc-014a-11e8-996f-afcad8879f8a.png)

Reply:
Right now product description works good for me, I don’t know why I couldn’t see your previous changes. 
According to point 3, please compare your local version to test version. Problem is on some resolutions about 1000px width. On our current version we have 2 products in row, on your version is there 3 products in row. Please compare also similiar products on product page on lower resolutions. This is big change so we have a lot of change requests :-)